files/tasks.py

```
@@ -413,176 +414,195 @@ def encode_media(
+        output_name_with_vtt_ending = f"{output_name}.vtt"
+        wav_file = f"{tmpdirname}/{subtitle_name}.wav"
+
+        logger.info(f"Video file path: {video_file_path}")
+        logger.info(f"Output name: {output_name}")
+        logger.info(f"WAV file: {wav_file}")
+
+        cmd = f"{settings.FFMPEG_COMMAND} -i {media.media_file.path} -ar 16000 -ac 1 -c:a pcm_s16le {wav_file}"
+        logger.info(f"Running ffmpeg command: {cmd}")
+
+        try:
+            ret = subprocess.run(
```

Ruff (0.13.3)

`subprocess` call with `shell=True` identified, security issue

(S602)